recipe/recipe.yaml

```
@@ -23,7 +23,7 @@ outputs:
 
     build:
       script:
-        - cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DFASTJET_ENABLE_DEBUG=OFF -DFASTJET_ENABLE_ALLCXXPLUGINS=ON -DFASTJET_ENABLE_CGAL=ON -DFASTJET_USE_INSTALLED_SISCONE=ON -DFASTJET_ENABLE_PYTHON=OFF -S . -B build
+        - cmake "${CMAKE_ARGS}" -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DFASTJET_ENABLE_DEBUG=OFF -DFASTJET_ENABLE_ALLCXXPLUGINS=ON -DFASTJET_ENABLE_CGAL=ON -DFASTJET_USE_INSTALLED_SISCONE=ON -DFASTJET_ENABLE_PYTHON=OFF -DCGAL_CMAKE_EXACT_NT_BACKEND=BOOST_BACKEND -DCGAL_DISABLE_GMP=ON -S . -B build
```

```suggestion
        - cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX="${PREFIX}" -DFASTJET_ENABLE_DEBUG=OFF -DFASTJET_ENABLE_ALLCXXPLUGINS=ON -DFASTJET_ENABLE_CGAL=ON -DFASTJET_USE_INSTALLED_SISCONE=ON -DFASTJET_ENABLE_PYTHON=OFF -DCGAL_CMAKE_EXACT_NT_BACKEND=BOOST_BACKEND -DCGAL_DISABLE_GMP=ON -S . -B build
```

Given the conversation on [#general > ✔ Validation C/Fortran compatibility with CMAKE_ARGS @ 💬](https://conda-forge.zulipchat.com/#narrow/channel/457337-general/topic/.E2.9C.94.20Validation.20C.2FFortran.20compatibility.20with.20CMAKE_ARGS/near/524945021) I learned that

> `${CMAKE_ARGS}` will contain quotes, so adding more quotes around it (in bash, which isn't necessarily smart about this) has the potential to mess up escaping

can be problematic. I need to make a PR to the conda-forge maintainer docs to make this better documented.

---

Yeah I was always wondering about this especially since it's unquoted in windows.... But windows is a special place.